tests/ui/app_list.js

```diff
@@ -393,10 +392,11 @@ casper.test.begin('Test collection detail page for app tile expanded state.', {
         });
 
         // Visit a collection details page and check it's not expanded.
-        helpers.startCasper({path: '/feed/collection/top-games'});
-        helpers.waitForPageLoaded(function() {
-            test.assertDoesntExist('.app-list.expanded');
-            test.assertDoesntExist('.previews');
+        casper.thenOpen(helpers.makeUrl('/feed/collection/top-games'), function() {
+            helpers.waitForPageLoaded(function() {
+                test.assertDoesntExist('.app-list.expanded');
```

Comment:
Thanks for checking. This should stay since it's testing a regression where the expanded tiles would be shown on a collection detail page if you toggled it on any other page.